docs/GettingStarted

```diff
@@ -85,22 +87,23 @@ The currently supported web server are:
 * Unicorn 4.8+ and 5.1+
 * Passenger 5.0 (experimental)
 
-==== Installation
+==== Auto Instrumentation
 
 Add the tracer gem to your +Gemfile+:
 
     gem 'ddtrace', :source => 'http://gems.datadoghq.com/trace/'
 
-Now you can set your service name, simply creating an initializer file in your +config/+ folder:
+To enable the Rails auto-instrumentation, you must create an initializer file in your +config/+ folder:
 
     # config/initializers/datadog-tracer.rb
 
     Rails.configuration.datadog_trace = {
-      default_service: 'my-rails-app',
+      auto_instrument: true,
+      auto_instrument_redis: true,
+      default_service: 'my-rails-app'
     }
 
-If you're using \Rails 3 or higher, the auto-instrumentation will be automatically activated and no more configuration
-is required. Your application will be listed as +my-rails-app+ in your {dashboard}[https://app.datadoghq.com/trace].
+If you're using \Rails 3 or higher, your application will be listed as +my-rails-app+ in your {dashboard}[https://app.datadoghq.com/trace].
```

Comment:
This URL is very likely to change, let's avoid linking app-internal URL in the doc. Maybe `as my-rails-app` or `as my-rails-app in your service list` could do it.